tox.ini

```diff
@@ -1,34 +1,35 @@
 [tox]
 envlist = 
-    py{36,37,38,39,310}
+    py{37,38,39,310,311}
     flake8
 
 [testenv]
 passenv=HOME
 sitepackages = False
-setenv =
-    PYTHONPATH = {toxinidir}
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/test-requirements.txt
 commands =
-    pip install -U pip
-    py.test --cov-report=term-missing --cov=patch_rebaser tests
+    pytest --cov-report=term-missing --cov=patch_rebaser tests
 
 [testenv:flake8]
 passenv=HOME
 sitepackages = False
-basepython = python
-deps = flake8
 commands = flake8 patch_rebaser tests integration_tests
 
 [testenv:integration]
+allowlist_externals =
+    /usr/bin/find
+    /usr/bin/sudo
 commands =
     /usr/bin/find . -name '*.pyc' -delete  # Avoid issues with pytests conf file discovery
     /usr/bin/sudo docker build -t patch_rebaser_integration_tests .
     /usr/bin/sudo docker run patch_rebaser_integration_tests /bin/sh -c "pytest -k integration_tests integration_tests"  # Ensure we only run the integration tests
 
 [testenv:integration_podman]
 commands =
+allowlist_externals =
```

Comment:
"commands" should be under allowlist_externals like for the docker version above

Reply:
Good catch, thanks, and fixed!